netbox/extras/views.py

```diff
@@ -1057,16 +1057,14 @@ def get_required_permission(self):
     def get(self, request, module, name):
         module = get_report_module(module, request)
         report = module.reports[name]()
+        jobs = module.get_jobs(name)
 
-        object_type = ContentType.objects.get(app_label='extras', model='reportmodule')
-        report.result = Job.objects.filter(
-            object_type=object_type,
-            object_id=module.pk,
-            name=report.name,
+        report.result = jobs.exclude(
```

Comment:
Shouldn't this be `filter()`?
```suggestion
        report.result = jobs.filter(
```